doc/developer-guide/api/functions/TSContScheduleOnPool.en.rst

```diff
@@ -55,6 +55,69 @@ called and continuations that use them have the same restrictions. ``TS_THREAD_P
 are threads that exist to perform long or blocking actions, although sufficiently long operation can
 impact system performance by blocking other continuations on the threads.
 
+Example Scenarios
+=================
+
+Scenario 1 (no thread affinity info, different types of threads)
+----------------------------------------------------------------
+
+NO affine thread set, plugin is calling the API on thread "A", "A" is of type "ET_TASK", and plugin
+wants to schedule on a "ET_NET" type thread provided in "tp".
+
+In this situation, system sees there is no thread affinity information stored in "contp". It then
+checks whether the type of thread "A" is the same as provided in "tp", and sees that "A" is "ET_TASK",
```

Comment:
"tp," ..."ET_TASK," <== commas inside the quote. Same for next line's ET_NET's period (and "B."....)

Reply:
Don't think this matters, just different styles, and they're not real quotes, just emphasize it's referencing the variable name.